src/modules/windowwalker/app/Window Walker/Components/Window.cs

```diff
@@ -239,18 +239,18 @@ public bool TaskListDeleted
         }
 
         /// <summary>
-        /// Get a value indicating whether the app is a cloaked UWP app
+        /// Gets a value indicating whether get a value indicating whether the app is a cloaked UWP app
         /// </summary>
         public bool IsUWPCloaked
         {
             get
             {
-                return (this.IsWindowCloaked() && this.ClassName == "ApplicationFrameWindow");
+                return IsWindowCloaked() && ClassName == "ApplicationFrameWindow";
             }
         }
 
         /// <summary>
-        /// Determines whether the specified windows is the owner
+        /// Gets a value indicating whether determines whether the specified windows is the owner

```

Comment:
And this one is slightly different `Gets a value indicating whether determines whether`